src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/17.diff.sql

```diff
@@ -0,0 +1,31 @@
+﻿/****************************************************************************************
+Guidelines to create migration scripts - https://github.com/microsoft/healthcare-shared-components/tree/master/src/Microsoft.Health.SqlServer/SqlSchemaScriptsGuidelines.md
+This diff is broken up into several sections:
+ - The first transaction contains changes to tables and stored procedures.
+ - The second transaction contains updates to indexes.
+ - IMPORTANT: Avoid rebuiling indexes inside the transaction, it locks the table during the transaction.
+******************************************************************************************/
+
+SET XACT_ABORT ON
+
+BEGIN TRANSACTION
+GO
+
+-- Data Type INT = 56
+IF EXISTS (SELECT * FROM sys.sequences WHERE Name = 'WorkitemKeySequence' AND system_type_id = 56)
+    BEGIN
+
+        DECLARE @sql NVARCHAR(MAX);
+        DECLARE @newWorkitemKeyStartVal INT;
+
+        -- Daily average is at ~50. Hence we buffer 100 on top of the last workitem key sequence
+        SET @newWorkitemKeyStartVal = (NEXT VALUE FOR dbo.WorkitemKeySequence) + 100;
+
+        SET @sql = CONCAT(N'DROP SEQUENCE dbo.WorkitemKeySequence; CREATE SEQUENCE dbo.WorkitemKeySequence AS BIGINT START WITH ', STR(@newWorkitemKeyStartVal), 'INCREMENT BY 1 MINVALUE ' + STR(@newWorkitemKeyStartVal) + ' NO CYCLE CACHE 10000');
+        EXEC sys.sp_executesql @sql;
```

Comment:
Do we know that we have enough permission to execute or should we have to GRANT permission?

Reply:
"sp_executesql" requires the user to be a public member. Permission is more important for the script it is executing. In this case, DROP/CREATE a sequence.